src/projections_site/templates/crs.html.j2

```diff
@@ -17,14 +17,21 @@
 
     <h3>Area of use</h3>
     <p>{{ crs.area_of_use.name }}</p>
-
-    {# TODO:
-    <p>Geodetic CRS: {{ util.get_geodetic_crs(crs) }}</p>
-    <p>Datum: {{ util.get_datum_name(crs) }}</p>
     <p>Data Source: EPSG  {{ util.get_epsg_version_string() }}</p>
+    <p>Datum: {{util.get_datum(crs)}}</p>
     <p>Coordinate System: {{ util.get_coordinate_system(crs) }}</p>
     <p>Projection: {{ util.get_coordinate_operation(crs) }}</p>
-    #}
+  {#
+ #<p>Coordinate System: {{ util.get_coordinate_system(crs) }}</p>
+#<p>Geodetic CRS: {{ util. get_geodetic_crs(crs) }}</p>
+#    <p>Datum: {{ util.get_datum_name(crs) }}</p>
+#    <p>Data Source: EPSG  {{ util.get_epsg_version_string() }}</p>
+#    <p>Coordinate System: {{ util.get_coordinate_system(crs) }}</p>
+#    <p>Projection: {{ util.get_coordinate_operation(crs) }}</p>
+#}
```

Comment:
I think all of these comments except geodetic CRS are done now. Do you think you can also implement geodetic CRS so we can clean up all these comments? Otherwise, let's just leave the geodetic CRS comment.

Reply:
yes, I can implement geodetic crs. I wanted to make sure i understand what it is first, thanks!

Reply:
@mfisher87 thank you for reviewing the changes.  Could you tell me the workflow after your review and suggested changes? I see additional commits.  if you can help me with the next step, that would be great. thanks!

Reply:
Nice! Looks like you accepted my suggested change, and all that's left is fixing the errors from pre-commit jobs :) If you have any questions about those, I'm happy to help out!